tools/StaticAnalysis/Exceptions/Az.Network/BreakingChangeIssues.csv

```diff
@@ -1,3 +1,9 @@
-"AssemblyFileName","ClassName","Target","Severity","ProblemId","Description","Remediation"
-"Microsoft.Azure.PowerShell.Cmdlets.Network.dll","Microsoft.Azure.Commands.Network.NewAzFirewallPolicyNatRuleCommand","New-AzFirewallPolicyNatRule","0","1050","The parameter set '__AllParameterSets' for cmdlet 'New-AzFirewallPolicyNatRule' has been removed.","Add parameter set '__AllParameterSets' back to cmdlet 'New-AzFirewallPolicyNatRule'."
-"Microsoft.Azure.PowerShell.Cmdlets.Network.dll","Microsoft.Azure.Commands.Network.UpdateAzureRmExpressRouteGatewayCommand","Set-AzExpressRouteGateway","0","2150","A validate range has been added for parameter 'MinScaleUnits' for cmdlet 'Set-AzExpressRouteGateway'.","Remove the validate range from parameter 'MinScaleUnits'."
+AssemblyFileName,ClassName,Target,Severity,ProblemId,Description,Remediation
```

Comment:
Please take care of their formats, they should be surrounded by quotes.

Reply:
i guess my text editor automatically removed quotes. sorry